go/client/ui.go

```diff
@@ -846,6 +846,32 @@ func (l LoginUI) DisplayResetProgress(ctx context.Context, arg keybase1.DisplayR
 	return nil
 }
 
+func (l LoginUI) ExplainDeviceRecovery(ctx context.Context, arg keybase1.ExplainDeviceRecoveryArg) error {
```

Comment:
can you handle `arg.Name == ""`? prompt wouldn't really make sense as is. https://github.com/keybase/client/pull/17165/files#diff-d759c186442347fe38ee29952fd5174dR265

Reply:
Can you have a device without Description (ie. name)? I thought it's not possible.

Reply:
i don't think you can but it's a sanity check just like checking `Description != nil`